pom.xml

```diff
@@ -277,9 +388,25 @@
                     <groupId>org.apache.maven.plugins</groupId>
                     <artifactId>maven-surefire-plugin</artifactId>
                     <version>${surefire.version}</version>
+                    <configuration>
```

Comment:
Is it possible to remove this configuration from `tests/pom.xml` file? In order to not duplicating definitions ..

Reply:
Removed, thanks.